website/docs/commands/package.md

```diff
@@ -362,3 +362,27 @@ scala-cli package --msi --output path.msi Hello.scala
 * launcher-app
 * exit-dialog
 * logo-path
+
+## Using directives
+
+Instead of passing some of `package` options directly from bash, it is possible to do so with [using directives](/docs/guides/using-directives).
```

Comment:
```suggestion
Instead of passing the `package` options directly from bash, it is possible to pass some of them with [using directives](/docs/guides/using-directives).
```